### PR TITLE
Fix username input issues

### PR DESCRIPTION
**Pull request by Akshay-Srivatsan**
1. I changed the title of the username inputs to specify "Username" (so people hopefully don't paste a bunch of url protocol/query stuff around their actual username)
2. I added a handler to detect if the user pasted a url matching the website's profile page scheme and, if so, extract the username from it.

### CODE REVIEW

File: js/question.js [hunk 1]
```diff
@@ -192,7 +192,34 @@ function Question(name, type, required, label, max_length, prefix, slug) {
 
     input.keyup(handler).change(handler).click(handler);
     input.keyup(needs_save).change(needs_save);
-    input.keydown(prevent_prefix_edit).on('cut copy paste', prevent_prefix_edit);
+    input.keydown(prevent_prefix_edit).on('cut', prevent_prefix_edit);
+    input.on('paste', function(e) {
+        var regex;
+        var index;
```
Comment (jasonscharff) [inline js/question.js]: I can think about this a bit more/review this for real, but this whole index thing is a bit sketch. You should be able to specify the capture group

Comment (jasonscharff) [reply js/question.js]: thinking about this a bit more, you should just be able to use the last group in all of these rather than specifying the index.

Comment (VjunetXuuftofi) [reply js/question.js]: you could also just use non capturing groups for the stuff you don't need to keep

Comment (jasonscharff) [reply js/question.js]: @VjunetXuuftofi this is also true and definitely the way to do this (see my first comment)

Comment (VjunetXuuftofi) [reply js/question.js]: I became a regex master doing this; https://github.com/VjunetXuuftofi/ElectoralDataProject/blob/master/parser.py

**Review by jasonscharff — CHANGES_REQUESTED** [review]
clean up your regex stuff.

**Review by jasonscharff — APPROVED** [review]
:shipit:

**Pull request merged**